didyoumean/__init__.py

```diff
@@ -1,2 +1,4 @@
 """Empty file. Might grow in the future."""
-import didyoumean_api
+from didyoumean import didyoumean_api
+from didyoumean.didyoumean_internal import add_suggestions_to_exception
```

Comment:
Ok, for a minute, I thought that pep8 was trolling us by first complaining about:
 - didyoumean/__init__.py:3:72: W292 no newline at end of file
and then:
 - didyoumean/__init__.py:4:1: W391 blank line at end of file

Things are actually a bit more subtle as per: https://stackoverflow.com/questions/9768139/pep8-conflict-between-w292-and-w391
I feel bad for asking you to give you trouble for such small details but if you could fix this, that would be much appreciated. Otherwise, I'll disable the style checks for the time being.

Reply:
my bad, I did put two empty lines at the end...

By the way if you use pre-commit with black and flake8 all of this stuff is automated and tested before submitting a commit ;)